sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md

```diff
@@ -1,7 +1,18 @@
 # Release History
 
-## 5.3.0-beta.1 (Unreleased)
-
+## 5.3.0 (2020-10-12)
+### New Features
+- Add `clientOptions` to `EventHubClientBuilder` to support for setting user's application id in the user-agent property
```

Comment:
Did we add the hostname verification as well? Should that be called out here?

Reply:
It's new feature of azure-core-amqp and an internal API of `EventHubClientBuilder`.  So we (me and @conniey) decided not to put it in EventHubs change log.